modules/compiler/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+  id 'antlr'
+}
+
+dependencies {
+  antlr 'org.antlr:antlr4:4.9.2'
+  implementation 'com.google.guava:guava:33.3.1-jre'
+  api 'org.apache.groovy:groovy:4.0.24'
+  api 'org.apache.groovy:groovy-json:4.0.24'
```

Comment:
TODO: move some code from language-server into compiler and change these back to `implementation`